scripts/run-browser-tests.js

```diff
@@ -1,54 +1,69 @@
-const cp = require('child_process');
+// const cp = require('child_process');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
 const fs = require('fs');
 const AWS = require('aws-sdk');
 
-// Extract command-line arguments
-const args = process.argv.slice(2); // Remove first two elements (node and script path)
-
-// Assuming you're passing two arguments: arg1 and arg2
-const baseUrl = args[0];
-const spec = args[1];
-
-if (!(args[0] && args[1])) {
-    console.error("baseURL and spec file are required as arguments")
-    process.exit(1)
+const pkgs = [
+    "aws",
+    "azure",
+    "gcp",
+]
+
+const results = {
+    tests: 0,
+    passes: 0,
+    failures: 0,
+    start: 0,
+    end: 0,
+    duration: 0,
+    failedPages: [],
+    failedPageCount: 0,
 }
 
-cp.exec(`CYPRESS_BASE_URL="${baseUrl}" yarn run cypress run --headless --reporter cypress-multi-reporters --spec "./cypress/e2e/${spec}"`, processResults)
+const proms = pkgs.map(pkg => {
+    return exec(`npm run test-api-docs-json -- --pkg=${pkg} || true`).then((stdout, stderr) => {
+        console.log(stdout)
+        processJSON(stdout, stderr)
```

Comment:
I removed the async from this function since it is not needed. In an earlier iteration of what I was working on I had it structured as an async method, but no longer needs to be.